conf/pacific/cephadm/tier-1_3node_cephadm_bootstrap.yaml

```diff
@@ -41,3 +41,4 @@ globals:
       node4:
         role:
           - client
+          - rgw
```

Comment:
Not sure if we need to add this role unless dedicated by the test case.

Also, the filename is kind of misleading... it states 3 node environment but actually it has a 4 node test environment.

Reply:
Changes done. Could you please review?